omod/src/main/compass/sass/runReport.scss

```diff
@@ -29,4 +29,7 @@
 img.small {
   width: 24px;
   height: 24px;
+}
+.drop-down-list {
+  max-width: 350px;
```

Comment:
@icrc-toliveira rather than setting a fixed width, isn't there a style that simply would prevent overflowing?

Reply:
Hi @mks-d 
After taking a quick look into this, it seems the best way to get this result is by setting the width of the drop-down to 100%

![image](https://user-images.githubusercontent.com/70152448/122527312-1cbc7180-d013-11eb-8ef0-7ecc9014f07c.png)

I also tried to change overflow settings instead, 
However this only after the external container (div)

![image](https://user-images.githubusercontent.com/70152448/122527628-673dee00-d013-11eb-966a-aa6bfb69c233.png)